persistence/src/test/java/cz/muni/fi/pa165/HouseDaoTest.java

```diff
@@ -31,68 +39,59 @@
     public EntityManager em;
 
     @Autowired
-    public HouseDao houseDao;
+    private HouseDao houseDao;
 
-    private House h1;
-    private House h2;
+    private House house;
 
 
     @BeforeMethod
     public void createHouses() {
-
-        h1 = new House();
-        h2 = new House();
-        h1.setName("SCALA");
-        h2.setName("MORGAL");
-
+        house =  new House();
+        house.setName("SCALA");
+        house.setAddress("MORAVAK");
+        house.setHistory("Brief history of time");
+        house.setHauntingFrom(Calendar.getInstance().getTime());
 
     }
+
     @Test
     public void createHouseTest() {
-        assertThat(h1.getId()).isNull();
-        houseDao.create(h1);
-        assertThat(h1.getId()).isNotNull();
+        assertThat(house.getId()).isNull();
+        houseDao.create(house);
+        assertThat(house.getId()).isNotNull();
+
     }
 
     @Test
-    public void deleteHouseTest() {
-        houseDao.create(h1);
-        houseDao.delete(h1);
-
-        assertThat(houseDao.getByName("SCALA")).isNull();
+    public void updateTest(){
+        houseDao.create(house);
+        houseDao.update(houseDao.getByName("SCALA").setName("MORGAL"));
```

Comment:
nepouzivaj takto String, sprav si to v before metode  a potom sa na to odkaz, pozri ine testy